wp-content/themes/missilethreat/assets/_scss/components/_navigation.scss

```diff
@@ -0,0 +1,113 @@
+@use 'abstracts' as *;
+
+.site-nav {
+  z-index: 10;
+  order: 1;
+  margin-left: rem(-12px);
+}
+
+.primary-menu {
+  @extend %font-ui-text-md-semibold-mobile-nav;
+  position: absolute;
+  top: rem(72px);
+  display: flex;
+  flex-direction: column;
+  width: 100vw;
+  color: $color-text-light-190;
+  text-transform: uppercase;
+  background: $color-primary-blue-200;
+  visibility: hidden;
+  transition: clip-path 0.3s linear, visibility 0.3s linear;
+  clip-path: inset(0 0 100% 0);
+      
+  &.is-visible {
+    z-index: 15;
+    visibility: visible;
+    clip-path: inset(0);
+  }
+
+  a {
+    display: block;
+    padding: rem(16px);
+
+    &:hover {
+      color: $color-text-light-100;
+      text-decoration: underline;
+    }
+  }
+}
+
+.hamburger,
+.nav-close {
+  width: rem(48px);
+  height: rem(48px);
+  color: $color-primary-light-gray;
+
+  &:hover {
+    color: $color-text-light-100;
+  }
+}
+
+.hamburger {
+  z-index: 15;
+  order: 1;
+  background: none;
+  border: 0;
+
+  &.is-invisible {
+    z-index: -1;
+    display: none;
+  }
+}
+
+.nav-close {
+  z-index: -1;
+  display: none;
+  order: 1;
+
+  &.is-visible {
+    z-index: 15;
+    display: initial;
+    background: none;
+    border: 0;
+  }
+}
+
+@include breakpoint('large') {
+  .site-nav {
+    z-index: 15;
+    order: 2;
+    width: 100%;
+    margin: 0;
+  }
+
+  .primary-menu {
+    @include font-size(14px);
+    top: 0;
+    flex-direction: row;
+    justify-content: flex-end;
+    width: 100%;
+    font-weight: 600;
+    line-height: 1.14;
+    white-space: nowrap;
+    @include full-width-background(0);
```

Comment:
```suggestion
```

You can remove this full-width bg declaration since you're not using it. It doesn't look like it was defined earlier, so you can also remove the ::before & after statements. If I'm wrong and you do have pseudo-elements you need to undo, then the way to remove them is how you did it on lines 99-102.